gyp_xwalk

```diff
@@ -370,7 +369,12 @@ if __name__ == '__main__':
     # Enable Sync compositor by default
     args.append('-Ddisable_sync_compositor=0')
     # Enable webaudio by default
-    args.append('-Ddisable_webaudio_hrtf=0')
+    if enable_web_audio:
+      args.append('-Ddisable_webaudio_hrtf=0')
+      args.append('-Duse_openmax_dl_fft=1')
+    else:
+      args.append('-Ddisable_webaudio_hrtf=1')
+      args.append('-Duse_openmax_dl_fft=0')
```

Comment:
It's could be cleaner if we can use a consistent "disable_web_audio" to replace 'disable_webaudio_hrtf' and 'use_openmax_dl_fft' option.